### DISCUSSION TITLE
Принудительная смена языковой раскладки клавиатуры.

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#476] [Как это сделать?] [answered]
Есть задача сканировать штрихкод. Штрихкод содержит буквы в латинице. При считывании штрихкода на компьютере используется кириллица и в итоге считывается и записывается неправильный текст штрихкода. Как можно решить эту проблему?

**Comment by LostAlly** [comment, marked as answer]
Ваш сканер работает в режиме клавиатуры, со всеми вытекающими. Возможно есть утилиты которые позволяют различные языковые настройки для разных клавиатур задавать, может это можно сделать средствами ОС.

**Reply by ttm-alexeypolunin** [reply]
Да, тотум здесь ничего не сделает — у браузера нет полномочий менять раскладку системы.
